Packs/ReversingLabs_TitaniumScale/Integrations/ReversingLabsTitaniumScale/README.md

```diff
@@ -1,1555 +1,853 @@
+                        {
+                            "classification": 3,
+                            "factor": 3,
+                            "ignored": false,
+                            "name": "TitaniumCore RHA1",
+                            "rca_factor": 8,
+                            "result": "Win32.Downloader.Unruy",
+                            "type": "internal",
+                            "version": "5.0.1.26"
+                        },
+                        {
+                            "classification": 3,
+                            "factor": 1,
+                            "ignored": false,
+                            "name": "TitaniumCore Machine Learning",
+                            "rca_factor": 6,
+                            "result": "Win32.Malware.Heuristic",
+                            "type": "internal",
+                            "version": "5.0.1.26"
+                        },
+                        {
+                            "classification": 0,
+                            "factor": 0,
+                            "ignored": false,
+                            "name": "drweb",
+                            "rca_factor": 0,
+                            "result": "Win32.HLLC.Asdas.7",
+                            "type": "av"
+                        },
+                        {
+                            "classification": 0,
+                            "factor": 0,
+                            "ignored": false,
+                            "name": "vba32",
+                            "rca_factor": 0,
+                            "result": "SScope.TrojanInjector.MY",
+                            "type": "av"
+                        },
+                        {
+                            "classification": 0,
+                            "factor": 0,
+                            "ignored": false,
+                            "name": "endgame",
+                            "rca_factor": 0,
+                            "result": "malicious (high confidence)",
+                            "type": "av"
+                        },
+                        {
+                            "classification": 0,
+                            "factor": 0,
+                            "ignored": false,
+                            "name": "ahnlab",
+                            "rca_factor": 0,
+                            "result": "Trojan/Win32.Kazy.R3559",
+                            "type": "av"
+                        },
+                        {
+                            "classification": 0,
+                            "factor": 0,
+                            "ignored": false,
+                            "name": "antivir",
+                            "rca_factor": 0,
+                            "result": "detected",
+                            "type": "av"
+                        },
+                        {
+                            "classification": 0,
```

Comment:
I generally avoid edditing the readme file by hand because our apps are getting huge. Most text editors are having memory issues with reading and editing our readmes. I think it would be pretty difficult to significantly cut down the context examples and still keeping some sense in the data.